src/docbuilder/queue.rs

```diff
@@ -29,13 +28,15 @@ impl DocBuilder {
                     krate.name, krate.version, err
                 ),
             }
-
-            add_count += 1;
         }
 
         index.set_last_seen_reference(oid)?;
 
-        Ok(add_count)
+        // Get the size of the queue
+        let query = conn.query("SELECT COUNT(*) as size FROM queue", &[])?;
+        let queue_size = query.iter().next().unwrap().get::<_, i64>("size") as usize;
+
```

Comment:
This doesn't look right, it returns the _total_ number of crates in the queue, not the number added.

Reply:
Yes, I changed it because that's what the comment [here](https://github.com/rust-lang/docs.rs/blob/822026308ff8250fa3f20ee374fc2d839d9c56de/src/docbuilder/queue.rs#L12) says it's supposed to do

Reply:
Are you sure the comment matches up with how the function is used? I don't think a behavior change like this belongs in the same PR.